README.md

```diff
@@ -79,11 +79,23 @@ This initiates Webpack, enabling the user to visualize the module online. Direct
 
 ### Testing
 
-To run tests: 
+To run tests:
 
 ```sh
 npm run test
 ```
 
+### CRUD Functionality
+
+GET Request -- send requests to '/api/moreplacestostay'. As currently written, the server will respond will all places and you will have to sort on front-end. This will be fixed in later versions of SDC as its untenable.
```

Comment:
@rogen-code 
Just a couple typos: As currently written, the server will respond **_will_** ( -> with) all places and you will have to sort on front-end. This will be fixed in later versions of SDC as **_its_** (-> it's) untenable.